src/components/SQForm/SQFormMultiValue.tsx

```diff
@@ -115,6 +147,7 @@ function SQFormMultiValue({
   React.useEffect(() => {
     // Don't run this effect every time displayOptions changes
     // This also prevents infinite rerenders
+    // Maybe use deep-equal? This won't work if key order changes
```

Comment:
Was this comment left in intentionally? Seem like it might be a note for you.

Reply:
ya, as a note in case we run into issues later on related to this `useEffect`.